command/rm.go

```diff
@@ -77,6 +78,12 @@ func (d Delete) Run(ctx context.Context) error {
 		printError(d.fullCommand, d.op, err)
 		return err
 	}
+	if !hasSameBuckets(srcurls) {
```

Comment:
I think we should not do this validation in  `command.Run()`. Like other validations, it should be a part of `command.Before()` (validateRMCommand).